xarray/core/rolling.py

```diff
@@ -669,7 +671,10 @@ def _array_reduce(
         if (
             OPTIONS["use_bottleneck"]
             and bottleneck_move_func is not None
-            and not is_duck_dask_array(self.obj.data)
+            and (
+                not is_duck_dask_array(self.obj.data)
+                or module_available("dask", "2024.11.0")
+            )
```

Comment:
can you delete the comment below please.

Reply:
Sorry, thanks!